shared/chat/conversation/messages/wrapper/wrapper-timestamp/index.js

```diff
@@ -71,7 +71,7 @@ class _WrapperTimestamp extends React.Component<Props & OverlayParentProps, Stat
     return (
       <Box style={styles.container}>
         {props.orangeLineAbove && <Box style={styles.orangeLine} />}
-        {!!props.timestamp && <Timestamp timestamp={props.timestamp} />}
+        {!!props.timestamp && !props.orangeLineAbove && <Timestamp timestamp={props.timestamp} />}
```

Comment:
better to just not set timeststamp if there's an orange line here: https://github.com/keybase/client/blob/master/shared/chat/conversation/messages/wrapper/wrapper-timestamp/container.js#L50